tools/tests/package-tests.js

```diff
@@ -546,11 +532,9 @@ selftest.define("sync local catalog", ["slow", "net", "test-package-server"],  f
 // `packageName` should be a full package name (i.e. <username>:<package
 // name>), and the sandbox should be logged in as that username.
 var createAndPublishPackage = selftest.markStack(function (s, packageName) {
-  var run = s.run("create", "--package", packageName);
-  run.waitSecs(20);
-  run.expectExit(0);
+  s.createPackage(packageName, "package-of-two-versions");
```

Comment:
We are still testing it in package-tests.js and publish.js. I actually made sure to check that before eliminating it in most other places.